### DISCUSSION TITLE
How to enable emmet/snippet with vite?

### DISCUSSION THREAD

**Discussion by fahmiegerton** [vuejs/language-tools#692] [Q&A] [answered]
Hi, I'm very new on vite and volar. I was using vetur and quite helpful with bunch of intellisense/emmet/snippet. Now that vite recommend using volar, I try to install and use it. But I couldn't see any intellisense/emmet/snippet appear when I type, for example, `h1`. Is there any way to set up or does volar didn't support it yet? thanks!

Edit: I'm using vue 3 generated by `yarn create vite`

**Comment by ghost** [comment, marked as answer]
(I'm using freshly installed VS Code 1.62 with just Volar + the yarn create vite **with typescript though** template you mentioned)
> IntelliSense when I type [...] `h1`

Seems to work for me:
![image](https://user-images.githubusercontent.com/5359825/141473761-f13eaaa2-eba2-4fdb-8859-185fe51c8b06.png)

Are you sure you're trying to type `h1` within the `<template>` tag?
Works fine for https://github.com/nuxt/framework/discussions/845#discussioncomment-1467028

----

> Snippets

If you mean Volar snippets from the `.vscode/volar` folder, see https://github.com/nuxt/framework/discussions/845#discussioncomment-1468494 and [vscode extension readme](https://github.com/johnsoncodehk/volar/blob/647391f6d72ba7c6d2154c541d5151a0e11b5364/extensions/vscode-vue-language-features/README.md?plain=1#L109)

**Reply by daguitosama** [reply]
It dosen't work for me in a Nuxt 3 project
![Screenshot from 2022-05-04 18-35-58](https://user-images.githubusercontent.com/34744883/166836541-f2e01ca0-34fe-4469-bd1c-76193941fcee.png)

**Reply by HenrijsS** [reply]
I'm still wondering why, after all this time, the snippets still aren't integrated into Volar.

**Reply by johnsoncodehk** [reply]
@d4g0 Can you reproduce this problem in https://github.com/johnsoncodehk/volar-starter?

**Reply by daguitosama** [reply]
Don't know exactly how, but this issue disappear once i update node and vs-code. So i recomend doing so and test that first for anybody

**Comment by Mahaveer1013** [comment]
I got the same issue now
